sql/parser/builtins.go

```diff
@@ -27,12 +27,14 @@ import (
 	"math"
 	"math/rand"
 	"reflect"
+	"strconv"
 	"strings"
 	"time"
 )
 
 var errEmptyInputString = errors.New("the input string should not be empty")
 var errAbsOfMinInt64 = errors.New("abs of min integer value (-9223372036854775808) not defined")
+var errRoundNumberDigits = errors.New("number of digits should be between 0 and 50")
```

Comment:
The new round error follows the style of `errEmptyInputString`, so I would leave it or change both. WDYT?